Step04/app/src/test/java/net/agilepartner/workshops/cqrs/app/End2EndTests.java

```diff
@@ -14,7 +14,7 @@
 	public void run() {
 		Repository<InventoryItem> repository = new InMemoryRepository<InventoryItem>();
 
-		CommandResolver resolver = InMemoryCommandResolver.getInstance();
+		CommandResolver resolver = new InMemoryCommandResolver();
```

Comment:
Sure. Why not.

Reply:
I did this change because Spring ensure that each bean it manages, is a singleton. 
But in this case, I think it will be used in actor modeling in the next steps. So, it make sense to keep it as a singleton.